Berksfile

```diff
@@ -3,5 +3,8 @@ source 'https://supermarket.chef.io'
 
 metadata
 
-cookbook 'apt'
-cookbook 'yum'
+group :integration do
+  cookbook 'apt'
+  cookbook 'yum'
```

Comment:
so, taking out all the references to apt and yum, and doing a kitchen destroy and kitchen converge got me 
```
Recipe: git::package
  * git_client[default] action install
    * apt_package[default :create git] action install
      * No candidate version available for git
      ================================================================================
      Error executing action `install` on resource 'apt_package[default :create git]'
      ================================================================================

      Chef::Exceptions::Package
      -------------------------
      No candidate version available for git
```